src/internal/markers.ts

```diff
@@ -84,14 +85,20 @@ export type ZipMarker = MarkerBase<MarkerType.ZIP> & {
   readonly sources: (Marker | null)[];
 };
 
+export type DebounceTimeMarker = MarkerBase<MarkerType.DEBOUNCE_TIME> & {
+  readonly sources: (Marker | null)[];
```

Comment:
```suggestion
  readonly sources: [(Marker | null)];
```
This change guarantees that once TS realizes the marker is a debounce marker, it will also know the array must have an element.

Reply:
Aha, I missed that from your previous suggestion, my bad